src/mqt/predictor/ml/helper.py

```diff
@@ -11,7 +11,16 @@
 from joblib import dump
 from qiskit import QuantumCircuit
 
-from mqt.predictor import utils
+from mqt.predictor import ml, utils
+
+
+def compile_prediction(qc: QuantumCircuit):
```

Comment:
small nitpick: wouldn't a more appropriate name for this function be `predict_and_compile`? (Since you first predict, and then compile).

Reply:
Changed it to be `qcompile` to have a shared interface for both predictors.